pkg/controllers/user/alert/common/common.go

```diff
@@ -19,3 +19,7 @@ func GetAlertManagerSecretName(appName string) string {
 func GetAlertManagerDaemonsetName(appName string) string {
 	return fmt.Sprintf("alertmanager-%s", appName)
 }
+
+func GetProjectAlertDisplayName(projectDisplayName, projectID string) string {
```

Comment:
I think cluster id should have display name too.

Reply:
updated